### DISCUSSION TITLE
Publish latest typing speed on GitHub personal account

### DISCUSSION THREAD

**Discussion by thevedicdeveloper** [monkeytypegame/monkeytype#3629] [Polls]
GitHub is turning into more of a personal account where developer showcases their hobbies, interests and achievements.

It would be great if we can publish our latest typing speed result on our GitHub user profile.

**Comment by Symbreaker** [comment]
Well, if it isn't prompting every user to do so and being too in your face somewhere

**Comment by Miodec** [comment]
https://github.com/search?q=monkeytype-readme&type=repositories

**Comment by ridemountainpig** [comment]
Hi @thevedicdeveloper , I have a project is MonkeyType Readme, it is a tool that transforms your MonkeyType typing data into SVGs, which you can put in your GitHub Readme or other place with image to show off your typing speed and accuracy.

<a href="https://monkeytype.com/profile/ridemountainpig">
   <img src="https://monkeytype-readme.repl.co/generate-svg/ridemountainpig/nord_light?lbpb=true" alt="My Monkeytype profile" />
  </a>

MonkeyType Readme: https://monkeytype-readme.repl.co/
MonkeyType Readme Github: https://github.com/ridemountainpig/monkeytype-readme
